src/server.rs

```diff
@@ -951,15 +951,10 @@ mod tests {
                 .to_request(),
         )
         .await;
-        assert_eq!(StatusCode::INTERNAL_SERVER_ERROR, response.status());
+        assert_eq!(StatusCode::CONFLICT, response.status());
```

Comment:
Do you think it makes sense to also update the related test fixtures to assert the status codes?

Reply:
yep, I thought about it. but curl does not print out the status code without having to parse either the verbose output or the result of `-I`.

Although I remember there was some way. I have to check my notes. Maybe I can find it.

Reply:
I'll look into it later tonight and add it to the fixtures.

Reply:
done